Logseq-linux-x64-0.6.6.AppImage fails on startup

### What happened?

Have been using `Logseq-linux-x64-0.6.1.AppImage` for a while, saw the 0.6.6 announcement.  Started it and got
```
A JavaScript error occurred in the main process
Uncaught Exception:
Error: libssl.so.1.1: cannot open shared object file: No such file or directory
    at process.func [as dlopen] (node:electron/js2c/asar_bundle:5:1800)
    at Object.Module._extensions..node (node:internal/modules/cjs/loader:1170:18)
    at Object.func [as .node] (node:electron/js2c/asar_bundle:5:1800)
    at Module.load (node:internal/modules/cjs/loader:982:32)
    at Module._load (node:internal/modules/cjs/loader:823:12)
    at Function.c._load (node:electron/js2c/asar_bundle:5:13331)
    at Module.require (node:internal/modules/cjs/loader:1006:19)
    at require (node:internal/modules/cjs/helpers:93:18)
    at Object.<anonymous> (/tmp/.mount_LogseqYqUNRT/resources/app/node_modules/@andelf/rsapi/index.js:159:31)
    at Module._compile (node:internal/modules/cjs/loader:1110:14)
libva error: /usr/lib/x86_64-linux-gnu/dri/i965_drv_video.so init failed
[790751:0418/031343.258961:ERROR:sandbox_linux.cc(374)] InitializeSandbox() called with multiple threads in process gpu-process.
```

Ubuntu up-to-date pre-22.04 (which shouldn't matter that much, the dependencies come from inside the AppImage.) A single graph, which continues to work fine in 0.6.1.

### Reproduce the Bug

Download and run https://github.com/logseq/logseq/releases/download/0.6.6/Logseq-linux-x64-0.6.6.AppImage and see the traceback above.

### Expected Behavior

what 0.6.1 did - run and display my graph.

### Screenshots

_No response_

### Desktop Platform Information

Ubuntu pre-22.04 (dist-upgrade as of 2022-04-18.)

### Mobile Platform Information

_No response_

### Additional Context

_No response_

On Ubuntu 22.04, I have
`Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.
A JavaScript error occurred in the main process
Uncaught Exception:
Error: Cannot find module '/shadow.js.shim.module$path.js'
Require stack:
- /tmp/.mount_LogseqvtD43E/resources/app/electron.js
- 
    at Module._resolveFilename (node:internal/modules/cjs/loader:934:15)
    at Function.n._resolveFilename (node:electron/js2c/browser_init:249:1128)
    at Module._load (node:internal/modules/cjs/loader:779:27)
    at Function.c._load (node:electron/js2c/asar_bundle:5:13331)
    at Module.require (node:internal/modules/cjs/loader:1006:19)
    at require (node:internal/modules/cjs/helpers:93:18)
    at /tmp/.mount_LogseqvtD43E/resources/app/electron.js:10370:40
    at Object.<anonymous> (/tmp/.mount_LogseqvtD43E/resources/app/electron.js:14974:3)
    at Module._compile (node:internal/modules/cjs/loader:1110:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1140:10)
libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)
[37988:0421/142417.885117:ERROR:sandbox_linux.cc(374)] InitializeSandbox() called with multiple threads in process gpu-process.
`

--- a/resources/package.json
+++ b/resources/package.json
@@ -36,7 +36,7 @@
     "https-proxy-agent": "5.0.0",
     "@sentry/electron": "2.5.1",
     "posthog-js": "1.10.2",
-    "@logseq/rsapi": "0.0.9",
+    "@logseq/rsapi": "0.0.11",
     "electron-deeplink": "1.0.9"
   },
   "devDependencies": {

